src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();

```

Comment:
```suggestion
    private final Random random = new Random();
```
i'm not sure Random is immutable. static final doesn't necessarily mean a constant (it's debatable). We usually use CONSTANT_CASE for immutable things
Better think of it like a dependency injection. no need for static